tests/topotests/pim_igmp_vrf/test_pim_vrf.py

```diff
@@ -339,7 +339,7 @@ def _test_vrf_pimreg_interfaces():
     assert res is None, assertmsg
 
     r1.vtysh_cmd("conf\ninterface red\nip pim")
-    r1.vtysh_cmd("conf\nvrf red\nip pim rp 192.168.0.12 239.100.0.1/32\nexit-vrf")
+    r1.vtysh_cmd("conf\nrouter pim vrf red\nrp 192.168.0.12 239.100.0.1/32")
```

Comment:
same comment above.

Reply:
Actually no longer exists. I'm now using one format string with placeholders for "red" and "blue" and the IPs.